jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -471,44 +471,21 @@ function assign_server_location(){
 #   value: "${KIE_SERVER_PORT}"
 #
 function configure_server_location() {
-    # KIE_SERVER_LOCATION matches our env-to-property naming convention,
-    # but KIE_SERVER_URL kept here for backwards compatibility
-    local location="${KIE_SERVER_LOCATION:-$KIE_SERVER_URL}"
-    if [ -z "${location}" ]; then
-        local protocol="${KIE_SERVER_PROTOCOL,,}"
-        local host="${KIE_SERVER_HOST}"
-        local defaultInsecureHost="${HOSTNAME_HTTP:-${HOSTNAME:-localhost}}"
-        local defaultSecureHost="${HOSTNAME_HTTPS:-${defaultInsecureHost}}"
-        local port="${KIE_SERVER_PORT}"
-        local path="/services/rest/server"
-        local routeName="${KIE_SERVER_ROUTE_NAME}"
```

Comment:
the envs configuration above must be kept for backwards capabilities.

Reply:
ok